plugins/common/functions

```diff
@@ -114,23 +114,38 @@ dokku_container_log_verbose_quiet() {
 }
 
 verify_app_name() {
-  local APP="$1"
+  local APP="$1"; local TAG="$2"
   [[ ! -n "$APP" ]] && dokku_log_fail "(verify_app_name) APP must not be null"
   [[ ! -d "$DOKKU_ROOT/$APP" ]] && dokku_log_fail "App $APP does not exist"
-  IMAGE="dokku/$APP"
+
+  if [[ -n "$TAG" ]]; then
```

Comment:
Since a tag will be applied to a docker image, I'm letting docker do that for us.